docs/data-guidelines.md

```diff
@@ -256,3 +257,26 @@ If the table indicates an initial version of "1" and an information source says
 If you're adding new data for Node.js, use `0.10.0` or later. If a feature was added in a version before `0.10.0`, use `0.10.0` anyway.
 
 This guideline was proposed in [#6861](https://github.com/mdn/browser-compat-data/pull/6861).
+
+## Mixins
+
+Mixins appear in WebIDL and in specifications to define Web APIs. For web developers, they aren't observable directly; they act as helpers to avoid repeating API definitions. BCD doesn't expose mixins.
```

Comment:
Well, we _do_ expose some mixins. Maybe it'll help if we give an imperative. Something like…

```suggestion
Mixins appear in WebIDL and in specifications to define Web APIs. For web developers, they aren't observable directly; they act as helpers to avoid repeating API definitions. Don't add mixins to BCD where they do not already exist.
```